editor/src/clj/editor/cljfx_form_view.clj

```diff
@@ -1263,14 +1262,42 @@
   (when (= KeyCode/ESCAPE (.getCode event))
     {:set-ui-state (assoc ui-state :filter-term "")}))
 
-(defn- filter-text-field [{:keys [text]}]
-  {:fx/type fx.text-field/lifecycle
-   :id "filter-text-field"
-   :style-class ["text-field" "filter-text-field"]
-   :prompt-text "Filter"
-   :text text
-   :on-key-pressed {:event-type :filter-key-pressed}
-   :on-text-changed {:event-type :filter-text-changed}})
+(defmethod handle-event :navigate-sections [{:keys [fx/event ui-state sections selected-section-title]}]
+  (when (instance? KeyEvent event)
+    (let [^KeyEvent event event]
+      (when (= KeyEvent/KEY_PRESSED (.getEventType event))
+        (condp = (.getCode event)
+          KeyCode/DOWN
+          (do (.consume event)
+              (when-let [next-section (->> sections
+                                           (drop-while (complement #{selected-section-title}))
```

Comment:
Niggle: The set-as-a-function equality check is a cute Clojure-ism, but wouldn't `#(not= % selected-section-title)` be both more readable and efficient?